### ISSUE TITLE
Don't import implementation files from another package

### ISSUE THREAD

**Issue by martin-robert-fink** [miguelpruivo/flutter_file_picker#746]
Before creating an issue, make sure that you are on the latest `file_picker` version and that there aren't already any similar opened inssues. Also, check if it isn't described on the [Wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki), specially on [Troubleshooting](https://github.com/miguelpruivo/flutter_file_picker/wiki/Troubleshooting) page.

Also, sometimes a simple `flutter clean` and `flutter build` again with latest file_picker version, may end up by fixing cached issues, so I encourage you to first do so.

**Describe the bug**
In the generated_plugin_registrant.dart file, I get a warning from file_picker - Don't import implementation files from another package.  Everything works, but I'd let to get rid of the warning.

```
//
// Generated file. Do not edit.
//

// ignore_for_file: directives_ordering
// ignore_for_file: lines_longer_than_80_chars

import 'package:file_picker/src/file_picker_web.dart'; // <=== Warning here
import 'package:file_selector_web/file_selector_web.dart';

import 'package:flutter_web_plugins/flutter_web_plugins.dart';

// ignore: public_member_api_docs
void registerPlugins(Registrar registrar) {
  FilePickerWeb.registerWith(registrar);
  FileSelectorWeb.registerWith(registrar);
  registrar.registerMessageHandler();
}

```

here are the `pubspec.yaml` dependencies:

```
dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
  get: ^4.1.4
  file_selector: ^0.8.2
  file_selector_windows: ^0.0.2
  file_selector_macos: ^0.0.4
  file_selector_linux: ^0.0.2
  file_picker: ^3.0.2+2
  flutter_svg: ^0.22.0
  uuid: ^3.0.4
  universal_platform: ^1.0.0-nullsafety
  styled_widget: ^0.3.1

```

**Platform**
- [] Android
- [] iOS
- [] Web
- [] Desktop (Go)

**Platform OS version**
What version did it happen?

**How are you picking?**
```
Please, post the snippet here
```

**Details to reproduce the issue**
Provide all the details to reproduce the issue.

**Error Log**
Please, post the _full_ console log of your issue, if applicable.

**Screenshots and/or video**
If applicable, add screenshots or video to help explain your problem.

**Flutter Version details**
Please, post the output of your `flutter doctor -v`, preferably, while running the issued device/simulator.

**Additional context**

**Comment by miguelpruivo** [comment]
Hi, you shouldn't have this issue when using the latest version of the plugin. A fix was introduced for this with 3.0.2.

**Comment by martin-robert-fink** [comment]
Here is my pubspec. I’m still getting the issue:



Martin

**Comment by miguelpruivo** [comment]
@martin-robert-fink that's correct. I've published a fix with 3.0.3 today.

**Comment by martin-robert-fink** [comment]
Hi -

Got home.  Changed pubspec.yaml to ^3.0.3.  Problem appears worse…. See screenshot:

**Comment by martin-robert-fink** [comment]
Looks like I had to shutdown/restart VSCode.  All better now.

Thanks,
Martin

**Comment by rconnett** [comment]
Hi @miguelpruivo - This issue appears to have come back in version 4.0.0.

**Comment by miguelpruivo** [comment]
Fixed again with 4.0.1. Let me know if it's good for you now. Thank you!

**Comment by rconnett** [comment]
> Fixed again with 4.0.1. Let me know if it's good for you now. Thank you!

That's sorted it.  Thanks!

**Comment by martin-robert-fink** [comment]
Agreed, fixed for me too!  Thanks - and also thanks for the desktop version, now I don't need two plug-ins!
